File: UltraList-App/app/src/main/java/com/iuliocesars/ultralist/Modelos/Oferta.java
package com.iuliocesars.ultralist.Modelos;

import com.google.android.gms.maps.model.LatLng;

import java.io.Serializable;
import java.math.BigDecimal;
import java.sql.Timestamp;

/**
 * Created by IulioCesars on 15/05/2018.
 */

public class Oferta implements Serializable
{
    public int id_oferta;
    public int id_articulo;
    public String nombre;
    public String descripcion;
    public String categoria;
    public BigDecimal precio;
    public int usuario_agrego;
    public Timestamp fecha_agrego;
    public int usuario_modifico;
    public Timestamp fecha_modifico;
    public int estatus;
    public String image_path;
    public double lat;
    public double lng;



    public void SetUbicacion(LatLng ll)
    {
        this.lat = ll.latitude;
        this.lng = ll.longitude;
    }

    public LatLng GetUbicacion()
    {
        return new LatLng(lat, lng);
    }
}

File: wsUltraList/wsUltraList/Modelo/Entidades/Oferta.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace wsUltraList.Modelo
{
    public partial class Oferta : Entidad<Oferta>
    {
        public Oferta() { }

        public Oferta(Articulo articulo)
        {
            id_articulo = articulo.id_articulo;
            nombre = articulo.nombre;
            descripcion = articulo.descripcion;
            categoria = articulo.categoria;
            precio = articulo.precio;
            usuario_agrego = articulo.usuario_agrego;
            fecha_agrego = articulo.fecha_agrego;
            usuario_modifico = articulo.usuario_modifico;
            fecha_modifico = articulo.fecha_modifico;
            estatus = articulo.estatus;
            image_path = articulo.image_path;
            lat = articulo.lat;
            lng = articulo.lng;
        }

    }
}